recipes/libcufile/meta.yaml

```diff
@@ -0,0 +1,157 @@
+{% set name = "libcufile" %}
+{% set version = "1.5.0.59" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set extension = "tar.xz" %}  # [not win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 222c49b29afb544c69542c4e658b0a1bd01fd2306f2136ad6990ccfff482a097  # [linux64]
+  sha256: 4e8d2b525322be9ab6ae574bd1d0325a4bf9c6c3bbdaa1611b5b969b7b34b0c4  # [aarch64]
+  patches:
+    - Makefile.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+outputs:
+  - name: libcufile
+    build:
+      missing_dso_whitelist:
+        - "*/libcuda.so.*"                     # [linux]
+      run_exports:
+        - {{ pin_subpackage("libcufile", max_pin="x") }}
+    files:
+      - lib/libcufile*.so.*
+      - etc/cufile.json
```

Comment:
Not sure if it's already been checked, but does cuFile use the file put here as expected?

Reply:
This is a good question. Will ask

It's possible we would need to set [`CUFILE_ENV_PATH_JSON`]( https://docs.nvidia.com/gpudirect-storage/api-reference-guide/index.html#json-config-file ) for this to work. That could be done with activation script

Suppose a fair question would be whether we want to do this or whether users would rather maintain their own configuration file

Reply:
It looks like we don't need this file. So we dropped it to avoid confusion

We can always revisit this in the feedstock